core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -73,50 +78,87 @@
+                stroke-linecap="round"
+                stroke-width="16"
+                d="M80 80 L220 80"/>
+          <path fill="none"
+                stroke="#10301D"
+                stroke-dasharray="15,28"
+                stroke-linecap="round"
+                stroke-width="16"
+                ng-init="generatePathParameters()"
+                d="<[pathSvgParameters]>"/>
+        </svg>
+
+        <img ng-src="<[getStaticImageUrl('/general/collection_otter.svg')]>" class="collection-otter">
+
+        <svg width="100"
+             height="100"
+             xmlns="http://www.w3.org/2000/svg"
+             xmlns:xlink="http://www.w3.org/1999/xlink"
+             version="1.1"
+             ng-repeat="node in collection.getCollectionNodes()"
+             style="position: absolute;
+                    left: <[pathIconParameters[$index].left]>;
+                    top: <[pathIconParameters[$index].top]>;">
+          <defs>
+            <pattern id="image<[$index]>" patternUnits="userSpaceOnUse" height="100" width="100">
+              <circle cx="50"
+                      cy="50"
+                      r="50"
+                      fill="<[pathIconParameters[$index].thumbnailBgColor]>"/>
+              <image x="10"
+                     y="10"
+                     height="80"
+                     width="80"
+                     xlink:href="<[pathIconParameters[$index].thumbnailIconUrl]>"></image>
+            </pattern>
+          </defs>
+          <a xlink:href="<['/explore/' + node.getExplorationId()]>"
+             ng-mouseover="updateExplorationPreview(node.getExplorationId());
+                           setIconHighlight($index);"
+             ng-mouseleave="togglePreviewCard();
+                            unsetIconHighlight($index);">
+             <circle cx="50"
+                     cy="50"
+                     r="42"
+                     fill="url(#image<[$index]>)"
+                     stroke="#006553"
+                     stroke-width="2"/>
+             <circle ng-show="($index===activeHighlightedIconIndex)?true:false"
+                     cx="50"
+                     cy="50"
+                     r="45"
+                     id="<['highlight' + $index]>"
+                     fill="none"
+                     stroke="green"
+                     stroke-width="10"
+                     stroke-opacity="0.3"/>
+          </a>
+        </svg>
+
+        <div ng-if="collection" class="oppia-card-preview-panel hidden-sm hidden-xs">
 
           <md-card class="oppia-activity-summary-tile md-default-theme">
             <div class="title-section" style="background-color: <[collectionSummary.thumbnail_bg_color]>; z-index: 1;">
               <img ng-src="<[collectionSummary.thumbnail_icon_url]>" class="thumbnail-image">
             </div>
+            <div class="title-section-mask" style="padding: 5px;"></div>
+            <div ng-attr-section="<[isWindowLarge ? undefined : 'right-section']>"></div>
```

Comment:
Do you know why it's called "right-section"? It seems more like a bottom section to me, unless I'm misunderstanding.

If you think this is the case too, would you mind changing it throughout the codebase? Also I think it would be better to use a class (or ng-class) rather than an ng-attr if possible.